hw/ip/rv_dm/rtl/deps.f

```diff
@@ -6,7 +6,6 @@
 ${PRJ_DIR}/hw/ip/prim/rtl/prim_flop_2sync.sv
 ${PRJ_DIR}/hw/ip/prim/rtl/prim_fifo_async.sv
 ${PRJ_DIR}/hw/ip/prim/rtl/prim_fifo_sync.sv
-${PRJ_DIR}/hw/ip/prim/rtl/prim_clock_inverter.sv
```

Comment:
this file looks to be stale... are we still using this or can this be removed entirely?

Reply:
No, I don't think so. I've created https://github.com/lowRISC/opentitan/pull/2858 to get them removed.

Reply:
Thanks!